### ISSUE TITLE
Error evaluating constraint 2: can't evaluate pow'(0,0.6).

### ISSUE THREAD

**Issue by svigerske** [coin-or/Ipopt#205]
Issue created by migration from Trac.

Original creator: design@np-soft.ru

Original creation time: 2013-06-25 16:56:54

Assignee: ipopt-team

Version: 3.10

I have try to solve the problem
```
g3 0 1 0 	# problem st_e41
5 2 1 1 1
2 0
0 0 
4 0 0
0 0 0 0 	
0 0 0 0 0
9 1
0 0
0 0 0 0 0
C0
o0
o2
o2
v2
o5
o1
n1
v0
n2
o5
o1
n1
v3
n2
o2
o1
n1
v2
o5
o1
n1
o2
v1
o1
n1
o2
o1
n1
v0
o1
n1
v3
n2
C1
o0
o0
o0
o2
n-200
o5
v0
n0.6
o2
n-200
o5
v1
n0.6
o2
n-200
o5
v2
n0.6
o2
n-300
o5
v3
n0.6
O0 0
n0
r
1 0.1
4 0
b # variables ranges
0 0.5 1
0 0.5 1
0 0.5 1
0 0.5 1
0 593.778559847802 900.000000000001
k4
2
4
6
8
J0 4
0 0
1 0
2 0
3 0
J1 5
0 0
1 0
2 0
3 0
4 1
G0 1
4 1
```

And I have follow output
```
******************************************************************************
This program contains Ipopt, a library for large-scale nonlinear optimization.
 Ipopt is released as open source code under the Eclipse Public License (EPL).
         For more information visit http://projects.coin-or.org/Ipopt
******************************************************************************

NOTE: You are using Ipopt by default with the MUMPS linear solver.
      Other linear solvers might be more efficient (see Ipopt documentation).


This is Ipopt version 3.10.1, running with linear solver mumps.

Number of nonzeros in equality constraint Jacobian...:        5
Number of nonzeros in inequality constraint Jacobian.:        4
Number of nonzeros in Lagrangian Hessian.............:       10

Error evaluating constraint 2: can't evaluate pow'(0,0.6).
```

It seems IPOPT try to evaluate derivation in zero even low range of variable is strongly positive... If degree is less then 1 then I have this error 

IPOPT Ipopt 3.8.1 has no this problem.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-06-25 17:19:07

There seems to be no starting point set, in which case Ipopt chooses 0. This is a bad choice for this model.

Anyhow, I agree that a point outside the bounds is a bad choice for the starting point and I can project the 0 onto the bounds to avoid this.

I don't know why it worked with Ipopt 3.8.1, the code in AmplTNLP::get_starting_point looks equivalent to me, but maybe the starting point was project into bounds before computing the scaling factors back then.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-06-25 17:25:48

Fixed in 5fa7711283. Will be available in next release.

**Comment by svigerske** [comment]
Comment by @svigerske created at 2013-06-25 17:25:48

Resolution: fixed
